react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+const pokedex = [
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' }
+];
+
+class Pokedex extends React.Component {
+  constructor(props) {
+    super(props);
+  }
```

Comment:
Since you aren't doing anything with the `constructor` you don't need it at all here

Reply:
Delete the `constructor`, it is not needed in this case